signin_parser.rb

```diff
@@ -26,11 +26,12 @@ def sign_in(url = "", login = "", password = "", enter = "")
     @browser.fill_in('login', with: login)
     @browser.fill_in('password', with: password)
     @browser.click_button(enter)
+    @browser
   end
 
-  def parse(class_to_parse = "", radio = "")
-    @browser.choose(radio)
-    @browser.all(:css, '.' + class_to_parse).map do |elem|
+  def parse
```

Comment:
It should be called something like parse_teachers. Name of method should describe what it does.